### ISSUE TITLE
Do capabilities really need to be cleared for an anonymous login?

### ISSUE THREAD

**Issue by museun** [museun/twitchchat#74]
on https://github.com/museun/twitchchat/blob/426e87995f4afcf1ca0f4cd98d776ea9d0585af4/src/twitch/userconfig.rs#L168-L170 we clear the caps if someone is using an anonymous login. 

this removes tags from messages which anonymous users /can/ receive.

so we should determine which tags are valid for `justinfan1234`

**Comment by museun** [comment]
Twitch acks. all 3 capabilities when using an anonymous login;
```
Cap {
    capability: "twitch.tv/membership",
    acknowledged: true,
}
Cap {
    capability: "twitch.tv/tags",
    acknowledged: true,
}
Cap {
    capability: "twitch.tv/commands",
    acknowledged: true,
}
```

So, this logic should be removed

**Comment by museun** [comment]
Fixed in 62d76900d9d3229a97fd4a9653722b7dc7f7fcde
